build/build.yml

```diff
@@ -210,13 +192,6 @@ steps:
       ARTIFACTSERVICES_SYMBOL_ACCOUNTNAME: 'microsoft'
       ARTIFACTSERVICES_SYMBOL_USEAAD: 'false'
 
-- ${{ if eq(parameters.publish, 'true') }}:
```

Comment:
Let's keep this section for uploading symbols if we can, for the PR/CI pipeline we want symbols uploaded. When running from the internal pipeline publish is set to false so this would be avoided

Reply:
Do 1es templates direct us away from using PublishPipelineArtifact@1?

Reply:
Added this back!